package NYT.NCoreDump.NProto;

import "yt/core/misc/proto/error.proto";

////////////////////////////////////////////////////////////////////////////////

message TCoreInfo
{
    required int32 process_id = 1;
    required string executable_name = 2;
    optional uint64 size = 3;
    optional NYT.NProto.TError error = 4;
    required int32 core_index = 5;
    optional int32 thread_id = 6;
    optional int32 signal = 7;
    optional string container = 8;
    optional string datetime = 9;
}

////////////////////////////////////////////////////////////////////////////////
